src/actions/trackerActions.js

```diff
@@ -291,14 +291,15 @@ export const submitWorklogs = () => async (
       referrer: 'no-referrer',
       body: JSON.stringify(worklog)
     })
+      .then(response => response.json())
+      .then(data => {
+        dispatch(setPointAsSynced(data.id))
+        return data.id
+      })
   )
 
-  const responses = await Promise.all(promises)
-  const syncedIds = (await Promise.all(responses.map(res => res.json())))
-    .map(r => r.id)
-    .filter(id => id >= 0)
+  const syncedIds = (await Promise.all(promises)).filter(id => id >= 0)
```

Comment:
we don't need this variable, so we also don't need the filtering here (do it above, in the second `then`)
this line can be replaced by this:
```suggestion
  await Promise.all(promises))
```

Reply:
@mikx98 How should I compute the number of succesful promises ?

Reply:
aaaah, I thought that was the end of the function as github showed :smile: sorry, I see that syncedIds is actually used and you need it. forget my comment :)